Testing/SystemTests/lib/systemtests/stresstesting.py

```diff
@@ -22,23 +22,24 @@
 
 File change history is stored at: <https://github.com/mantidproject/systemtests>.
 '''
-import sys
-import os
-import types
-import re
-import time
 import datetime
-import platform
-import subprocess
-import tempfile
 import imp
 import inspect
-import abc
+import os
 import numpy
+import platform
+import re
+import subprocess
+import sys
+import tempfile
+import time
 import unittest
 
 # Path to this file
 THIS_MODULE_DIR = os.path.dirname(os.path.realpath(__file__))
+# Some windows paths can contain sequences such as \r, e.g. \release_systemtests
+# and need escaping to be able to add to the python path
+TESTING_FRAMEWORK_DIR = THIS_MODULE_DIR.replace('\\', '\\\\')
```

Comment:
Do you know about raw strings? This could be done without so much escaping as `...replace(r'\', r'\\')`